cfgov/jinja2/v1/complaint/complaint-landing.html

```diff
@@ -0,0 +1,323 @@
+          Consumer Response’s Annual Report to Congress</a>.
+      </p>
+      <p>
+        <a href="/complaint/data-use/">
+          More on how we collect and use the data
+        </a>
+      </p>
+    </div>
+    <div class="content-l_col content-l_col-1-3 db-numbers callouts-large">
+      <p class="callout"><span class="numeric-callout percent-timely">98%</span> of complaints sent to companies get timely responses</p>
+    </div>
+  </div>
+</div>
+<div class="block block__bg block__flush-top block__flush-bottom">
+  <div class="content-l">
+    <div class="content-l_col content-l_col-2-3">
+      <h3>
+        Past changes to the database
+      </h3>
+      <p>
+        In April 2017, we made some changes to the way products, sub-products, issues, and sub-issues are grouped. The Consumer Complaint Database shows the consumer’s original products, sub-products, issues, and sub-issues selections consistent with the options available on the form at the time the consumer submitted the complaint.
+      </p>
+      <ul class="m-list m-list__links">
+        <li class="m-list_item">
+          <a class="a-link a-link__icon" href="https://files.consumerfinance.gov/f/documents/201704_cfpb_Summary_of_Product_and_Sub-product_Changes.pdf">
+            <span class="a-link_text">Learn more about complaint form changes to products, sub-products, issues, and sub-issues</span>
+          </a>
+        </li>
+        <li class="m-list_item">
+          <a class="a-link a-link__icon" href="https://files.consumerfinance.gov/f/documents/201704_cfpb_Consumer_Complaint_Form_Product_and_Issue_Options.pdf">
+            <span class="a-link_text">View full list of complaint form products, sub-products, issues, and sub-issues</span>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+<div class="block block__border-top block__padded-top block__flush-top">
+  <div class="section-five">
+    <h2 id="download-the-data">Download the data</h2>
+    <p class="download-text">Our reports highlight some of what we’re seeing in the complaints we handle, but know there are more ways to use, analyze and build on the data. That’s why we make it possible for you to download the data, once we have taken steps to remove personal information.</p>
+    <p class="download-text">You can download all complaint data as either a CSV or JSON file here, or you can download a subset of the data—such as all complaints for a specific product—by filtering the full data set and exporting your results.</p>
+    <ul class="m-list m-list__links">
+      <li class="m-list_item">
+        <a class="a-link a-link__icon" href="http://files.consumerfinance.gov/ccdb/complaints.csv.zip">
+          <span class="a-link_text">Download all complaint data | CSV</span>
+        </a>
+      </li>
+      <li class="m-list_item">
+        <a class="a-link a-link__icon" href="http://files.consumerfinance.gov/ccdb/complaints.json.zip">
```

Comment:
same with this one too

Reply:
Changed in https://github.com/cfpb/consumerfinance.gov/pull/6357/commits/30aae2520a2af9a9c31c8f7330cc8f914f7d2aea